### ISSUE TITLE
Teo runtime build error

### ISSUE THREAD

**Issue by akelyasir** [teodevgroup/teo#63]
Hi @victorteokw 

As of today, it started giving errors during build.

```
error[E0275]: overflow evaluating the requirement `namespace::Namespace: std::fmt::Debug`
  --> teo-runtime/src/namespace/mod.rs:45:10
   |
45 | #[derive(Educe)]
   |          ^^^^^
   |
   = note: required for `BTreeMap<std::string::String, namespace::Namespace>` to implement `std::fmt::Debug`
   = help: see issue #48214
   = note: this error originates in the derive macro `Educe` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by victorteokw** [comment]
Hi @akelyasir, I will look into this.

**Comment by victorteokw** [comment]
Hi @akelyasir, could me know what schema do you use to generate the Rust entities?

**Comment by akelyasir** [comment]
Hi @victorteokw 

I installed it on the computer via teo-development-setup to fix the error that appeared in the file upload section and share it. It gave an error without doing anything. I didn't use any scheme. It was a clean installation, I directly downloaded the files to the computer from github.

Rust version information is as follows.

```
rustup 1.27.1 (54dd3d00f 2024-04-24)
info: This is the version for the rustup toolchain manager, not the rustc compiler.
info: The currently active `rustc` version is `rustc 1.78.0 (9b00956e5 2024-04-29)`
```

`cargo 1.78.0 (54d8815d0 2024-03-26)`

```
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/yasirakel/.rustup

installed toolchains
--------------------

stable-x86_64-unknown-linux-gnu (default)
nightly-x86_64-unknown-linux-gnu
1.78.0-x86_64-unknown-linux-gnu
1.75-x86_64-unknown-linux-gnu

installed targets for active toolchain
--------------------------------------

wasm32-unknown-unknown
x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.78.0 (9b00956e5 2024-04-29)
```

**Comment by victorteokw** [comment]
Hi @akelyasir, the upload error is fixed in version 0.2.36.

**Comment by victorteokw** [comment]
Hi @akelyasir, delete old files and rerun `bash <(curl -o- https://raw.githubusercontent.com/teocloud/teo-development-setup/main/setup.sh)`
I just cloned and `cargo build` on a new Mac. It builds fine without any errors.
